package.json

```diff
@@ -20,17 +20,18 @@
     "@types/node": "^10.11.7",
     "coveralls": "^3.0.2",
     "husky": "^1.1.2",
-    "jest": "^23.6.0",
+    "jest": "^28.1.3",
```

Comment:
Is this required? bcuz causing the CI to fail.

Reply:
@Shivansh-yadav13 upgrade node version in CI can resolve this

Reply:
@Zxilly which versions should we use in CI today?

Reply:
@hsluoyz It's ok.
In fact, we can create a `reuseable workflow` in casbin org to standardize test env.

Reply:
@Shivansh-yadav13 can you make another PR to update the CI to use Node.js [16, 17, 18] ?

Reply:
@Zxilly will all repos under: https://github.com/node-casbin use the same workflow?

Reply:
> @Shivansh-yadav13 can you make another PR to update the CI to use Node.js [16, 17, 18] ?

Yes.

Reply:
@hsluoyz yes

Reply:
Let's keep it first this way. Involving the org setting may bring trouble when we want to migrate repos between orgs one day.

Reply:
@Shivansh-yadav13 Jest drop support for node 15 at https://github.com/facebook/jest/releases/tag/v28.0.0